src/routes/trade/trade.module.scss

```
@@ -235,14 +244,16 @@
 }
 
 .container {
-  display: grid;
+  display: flex;
   padding: 0.5rem;
   box-sizing: border-box;
   margin-bottom: 1rem;
   justify-content: center;
   border: 1px solid var(--button-green);
```

Can we use `--border-1` here instead?
```suggestion
  border: 1px solid var(--border-1);
```

---

sure.Thanks